assets/src/components/site-scan-results/style.scss

```
@@ -97,3 +121,20 @@
 		fill: var(--amp-settings-color-brand);
 	}
 }
+
+.site-scan-results__urls-list {
+	margin: 1.5rem 0;
+	padding: 0 1rem;
+}
+
+.site-scan-results__source-detail {
+	background-color: #fff;
+	border: 1px solid #dedede;
+	border-radius: 5px;
+	padding: 15px;
+	max-height: 510px;
+	overflow: scroll;
+	font-size: 12px;
+	line-height: 2;
+	white-space: pre-wrap;
```

I think it's better to not do the wrapping:

```suggestion
	white-space: pre;
```